cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/compute/GcpInstanceResourceBuilder.java

```diff
@@ -72,6 +73,13 @@
         instance.setCanIpForward(Boolean.TRUE);
         instance.setNetworkInterfaces(getNetworkInterface(context.getNetworkResources(), location.getRegion(), group, compute, projectId));
         instance.setDisks(listOfDisks);
+        Scheduling scheduling = new Scheduling();
+        boolean preemptible = false;
+        if (template.getParameter("preemptible", Boolean.class) != null) {
```

Comment:
Could you create a constant from the "preemptible" string as seen on the top of the class, please? Otherwise LGTM.